[2025-09-19T14:44:40.599+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-09-19T14:44:40.631+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: mayo_pipeline.etl_vitals manual__2025-09-19T14:44:39.624469+00:00 [queued]>
[2025-09-19T14:44:40.638+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: mayo_pipeline.etl_vitals manual__2025-09-19T14:44:39.624469+00:00 [queued]>
[2025-09-19T14:44:40.639+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-09-19T14:44:40.654+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): etl_vitals> on 2025-09-19 14:44:39.624469+00:00
[2025-09-19T14:44:40.661+0000] {standard_task_runner.py:63} INFO - Started process 187 to run task
[2025-09-19T14:44:40.664+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'mayo_pipeline', 'etl_vitals', 'manual__2025-09-19T14:44:39.624469+00:00', '--job-id', '3', '--raw', '--subdir', 'DAGS_FOLDER/mayo_pipeline.py', '--cfg-path', '/tmp/tmphyw5j7ks']
[2025-09-19T14:44:40.666+0000] {standard_task_runner.py:91} INFO - Job 3: Subtask etl_vitals
[2025-09-19T14:44:40.725+0000] {task_command.py:426} INFO - Running <TaskInstance: mayo_pipeline.etl_vitals manual__2025-09-19T14:44:39.624469+00:00 [running]> on host 079e61bbc317
[2025-09-19T14:44:40.994+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='you' AIRFLOW_CTX_DAG_ID='mayo_pipeline' AIRFLOW_CTX_TASK_ID='etl_vitals' AIRFLOW_CTX_EXECUTION_DATE='2025-09-19T14:44:39.624469+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-09-19T14:44:39.624469+00:00'
[2025-09-19T14:44:40.995+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-09-19T14:44:41.023+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-19T14:44:41.025+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'cd /opt/project && python etl/flatten_mimic_vitals.py']
[2025-09-19T14:44:41.035+0000] {subprocess.py:86} INFO - Output:
[2025-09-19T14:44:41.097+0000] {subprocess.py:93} INFO - python: can't open file '/opt/project/etl/flatten_mimic_vitals.py': [Errno 2] No such file or directory
[2025-09-19T14:44:41.100+0000] {subprocess.py:97} INFO - Command exited with return code 2
[2025-09-19T14:44:41.101+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-09-19T14:44:41.112+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/models/baseoperator.py", line 401, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.11/site-packages/airflow/operators/bash.py", line 243, in execute
    raise AirflowException(
airflow.exceptions.AirflowException: Bash command failed. The command returned a non-zero exit code 2.
[2025-09-19T14:44:41.116+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=mayo_pipeline, task_id=etl_vitals, run_id=manual__2025-09-19T14:44:39.624469+00:00, execution_date=20250919T144439, start_date=20250919T144440, end_date=20250919T144441
[2025-09-19T14:44:41.133+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 3 for task etl_vitals (Bash command failed. The command returned a non-zero exit code 2.; 187)
[2025-09-19T14:44:41.158+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2025-09-19T14:44:41.167+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
